### PR TITLE
9/30

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2022-09-30.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+But as Dolt evolved into a [version controlled database](https://www.dolthub.com/blog/2022-08-04-database-versioning/), we searched for database metaphors to describe DoltHub. The best way to think about DoltHub in the database context is a user interface to manage snapshots (ie. backups) and, oh, snapshots are editable and merge-able. This was useful but we don't think it's what people want for a version controlled database. 
+
+People want a production database that they can clone, branch, merge, and diff. They want to make pull requests against the production database. The production database is the main branch AND the collaboration hub. Bug in production? Clone production to your laptop to debug. Schema change? Make a branch of production and test it. Pull request with peer review to get it merged. That's what [Hosted Dolt](https://hosted.doltdb.com/) is. It's a production database with Git-style versioning. It changes the game. It's just a better Hosted MySQL.
+
+[![Which Dolt is right for you?](../images/dolt-options.png)](https://hosted.doltdb.com/)
```
Comment (tbantle22) [inline emails/2022-09-30.md:25]: Want me to quickly make a link to that chart on the homepage? Can be `https://hosted.doltdb.com#which-dolt-is-right-for-you`?

File: emails/2022-09-30.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Dolt How To
+
+I've been in a bit of a How To mood lately. This week I piblished to How To blogs, [When to make a Dolt Commit](https://www.dolthub.com/blog/2022-09-28-when-to-dolt-commit/) and [How to Rollback in Dolt](https://www.dolthub.com/blog/2022-09-23-dolt-rollback-options/). This is on the heels of [Getting Started with Permissions](https://www.dolthub.com/blog/2022-08-19-dolt-permissions-introduction/) and my epic general
```
Comment (tbantle22) [inline emails/2022-09-30.md:29]:
```suggestion
I've been in a bit of a How To mood lately. This week I published two How To blogs, [When to make a Dolt Commit](https://www.dolthub.com/blog/2022-09-28-when-to-dolt-commit/) and [How to Rollback in Dolt](https://www.dolthub.com/blog/2022-09-23-dolt-rollback-options/). This is on the heels of [Getting Started with Permissions](https://www.dolthub.com/blog/2022-08-19-dolt-permissions-introduction/) and my epic general 
```

**Review by tbantle22 — APPROVED** [review]

File: emails/2022-09-30.md (hunk 1 above)
Comment (timsehn) [reply emails/2022-09-30.md:25]: That would be great.

Comment (tbantle22) [reply emails/2022-09-30.md:25]: Kk will be done before I leave today

**Pull request merged**